src/ansys/geometry/core/designer/design.py

```diff
@@ -1220,3 +1220,139 @@ def _update_design_inplace(self) -> None:
 
         # Read the existing design
         self.__read_existing_design()
+
+    def update_from_tracker(self, tracker_response):
+        """
+        Update the design with the changed bodies while preserving unchanged ones.
+
+        Parameters
+        ----------
+        changed_bodies : list[dict]
+            A list of dictionaries representing changed body information. Each dictionary
+            should contain 'id', 'name', and 'is_surface' keys.
+        """
+        print("Updating design using the tracker...")
```

Comment:
Use the logger rather than print statements

Reply:
I think the logs would be nice to have nonetheless. Maybe we can simplify them.

And yes, use self._grpc_client.log.** for performing logging, whatever log level message you want to put. This is client-side logging.

Some might be good at debug, others might be good at info. Thanks @umutsoysalansys!